IAteIt/IAteIt/View/MyProfile/MyProfileView.swift

```diff
@@ -10,31 +10,15 @@ import SwiftUI
 struct MyProfileView: View {
     
     let paddingLR: CGFloat = 16
-    let profileImgSize: CGFloat = 120
-    
     
     var body: some View {
-        VStack {
-            ZStack {
-                Circle()
-                    .frame(width: profileImgSize, height: profileImgSize)
-                Image("Sample_Profile1")
-                    .resizable()
-                    .scaledToFill()
-                    .layoutPriority(-1)
-            }
-            .clipped()
-            .cornerRadius(profileImgSize/2)
-
-            ScrollView(.vertical, showsIndicators: false) {
-                DailyMealCellView()
-
+        ScrollView {
+            VStack {
+                ProfileCellView()
+                MealListView()
             }
-            
         }
         .padding([.leading, .trailing], paddingLR)
```

Comment:
```suggestion
        .padding(.horizontal, .paddingHorizontal)
```
좌우 padding 16은 CGFloat extension 위와 같이 `.paddingHorizontal`로 사용하실 수 있습니다!

Reply:
넵 감사합니다!